### PR TITLE
BAU: Nepali translation update

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves N/A

**Overall change:**
Amends a Nepali translation per request from the Nepali editorial team.

**Code changes:**

- Updates translation
- Updates snapshots

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/nepali.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const service = {
     lang: `ne`,
     articleAuthor: `http://www.facebook.com/bbcnews`,
     articleTimestampPrefix: '',
-    articleTimestampSuffix: 'मा अद्यावधिक',
+    articleTimestampSuffix: ' अद्यावधिक',
```
Comment (chewu) [inline src/app/lib/config/services/nepali.js]: Is the space in front intended?

Comment (jimjohnsonrollings) [reply src/app/lib/config/services/nepali.js]: I believe so, otherwise we won't leave a space between the date and the word.

Comment (jimjohnsonrollings) [reply src/app/lib/config/services/nepali.js]: Although looking at the test maybe that's wrong...

Comment (jimjohnsonrollings) [reply src/app/lib/config/services/nepali.js]: I have removed the space.  Good spot.

**Review by chewu — APPROVED** [review]

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**